### ISSUE TITLE
spellchecker: don't complain about case

### ISSUE THREAD

**Issue by jsoref** [checkstyle/checkstyle#5732]
Right now the spell checker gets annoyed about changes like:
```
@@ -1189 +1189 @@
-microsoft
+Microsoft
```

While it is a change, the spell checker can tolerate it...

**Comment by romani** [comment]
fix is merged
